testing/tests/DevExpress.ui.widgets.editors/colorBox.markup.tests.js

```diff
@@ -35,12 +35,20 @@ QUnit.module('ColorBox', {
     });
 
     QUnit.test('Render color input', function(assert) {
-        const $colorBox = showColorBox.call(this); const $input = $colorBox.find('.' + COLOR_BOX_INPUT_CLASS);
+        const $colorBox = showColorBox.call(this);
+        const $input = $colorBox.find('.' + COLOR_BOX_INPUT_CLASS);
```

Comment:
[minor] - just to stay consistent
```suggestion
        const $input = $colorBox.find(`.${COLOR_BOX_INPUT_CLASS}`);
```

Reply:
It is not my code, I just move it to new line